### PR TITLE
books and membership implemented

### PR DESCRIPTION
**Pull request by talhakhalidmtk**
- Used simple jwt for authentication and rest_registration
- Used LimitOffset Pagination
- Created Book, Publisher, BookContributor, Rack, Membership Models

### CODE REVIEW

File: account/models.py
Comment (adeelaarbi) [inline account/models.py:1]: In Account models create custom user use it's email as username

File: loan/models.py
Comment (adeelaarbi) [inline loan/models.py:1]: It's not load it's lend

File: loan/views.py
Comment (adeelaarbi) [inline loan/views.py:1]: validations should be done in serializers

File: shelfstacker/__init__.py
Comment (adeelaarbi) [inline shelfstacker/__init__.py:1]: reanme shelfstacker to config

File: catalog/models.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        ('art', 'Art'),
+    )
+    title = models.CharField(max_length=70, help_text="The title of the book.")
+    isbn = models.CharField(max_length=13, unique=True,
+                            help_text="The ISBN number of the book.")
+    category = models.CharField(
+        max_length=20,
+        choices=CATEGORY_CHOICES,
+        help_text="The category of the book.")
+    rack = models.ForeignKey(Rack, on_delete=models.SET_NULL, null=True)
```
Comment (ihtram) [inline catalog/models.py:41]: @talhakhalidmtk  can you let me know the relation between Book and Rack? can a book belong to more than one racks?

Comment (talhakhalidmtk) [reply catalog/models.py:41]: @ihtram 
The relation between Book and Rack is One-to-Many. One Book can belong to only one Rack but one Rack may have many books. 
No, a book can't belong to more than one racks.

**Pull request merged**